tests/test_workflows.py

```diff
@@ -0,0 +1,79 @@
+import csv
```

Comment:
You don't need to include the cli runner fixture in any of these workflow tests since they aren't testing the cli.

Reply:
I was using it in 2 tests for the `.isolated_filesystem()` to avoid writing the output CSVs to the project directory. But I removed the others

Reply:
Ah, you don't need it for those either! The `sample_files_dir` fixture is already using pytest's `tmp_path` feature which will handle writing and removing those output files during tests.